compiler/src/pretyping.ml

```diff
@@ -1594,14 +1597,7 @@ let rec tt_instr pd asmOp (env : 'asm Env.env) ((annot,pi) : S.pinstr) : 'asm En
       let lvs, is = tt_lvalues pd env (L.loc pi) ls None tlvs in
       assert (is = []);
       let es  = tt_exprs_cast pd env (L.loc pi) args tes in
-      let is_inline = 
-        match Annot.ensure_uniq1 "inline" Annot.none annot with
-        | Some () -> E.InlineFun
-        | None -> 
-          match f.P.f_cc with 
-          | FInfo.Internal -> E.InlineFun
-          | FInfo.Export | FInfo.Subroutine _ -> E.DoNotInline in
-      let annot = Annot.consume "inline" annot in
+      let is_inline = P.is_inline annot f.P.f_cc in
```

Comment:
I think if the function is annotated with inline, then the pretyping should 
add the annotation to the call.
So the is_inline function can simply look the annotation.